packages/pwa-kit-create-app/program.json

```diff
@@ -472,6 +472,56 @@
                 },
                 "private": true
             },
+            {
```

Comment:
In an ideal world we'd use the new `--stdio` flag to pipe the configuration directly in to the generator. This means we don't have to change and release a new version of the create app project. But I think given the time constraint and the need for use to migrate this generator work to the `next` branch, and because you are making change in other parts of the generator, we can go ahead with this.

But in the future lets try to use that new feature, and ask ourselves if we see customers using these presets. If the answer is no, and we are only wanting one for some git hub actions, maybe we manage those externally.